src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -1186,4 +1186,28 @@
     <value>Shortcut</value>
     <comment>The label for a "key chord listener" control that sets the keys a key binding is bound to.</comment>
   </data>
-</root>
\ No newline at end of file
+  <data name="Appearance_TextFormattingHeader.Text" xml:space="preserve">
+    <value>Text Formatting</value>
+    <comment>Header for a control to how text is formatted</comment>
+  </data>
+  <data name="Appearance_IntenseTextStyle.Header" xml:space="preserve">
+    <value>Intense Text Format</value>
+    <comment>Header for a control to select how "intense" text is formatted (bold, bright, both or none)</comment>
+  </data>
+  <data name="Appearance_IntenseTextStyleNone.Content" xml:space="preserve">
+    <value>None</value>
+    <comment>An option to choose from for the "intense text format" setting. When selected, "intense" text will not be rendered differently</comment>
+  </data>
+  <data name="Appearance_IntenseTextStyleBold.Content" xml:space="preserve">
+    <value>Bold</value>
+    <comment>An option to choose from for the "intense text format" setting. When selected, "intense" text will be rendered as bold text</comment>
+  </data>
+  <data name="Appearance_IntenseTextStyleBright.Content" xml:space="preserve">
+    <value>Bright</value>
+    <comment>An option to choose from for the "intense text format" setting. When selected, "intense" text will be rendered in a brighter color</comment>
+  </data>
+  <data name="Appearance_IntenseTextStyleAll.Content" xml:space="preserve">
+    <value>Both</value>
+    <comment>An option to choose from for the "intense text format" setting. When selected, "intense" text will be rendered as both bold text and in a brighter color</comment>

```

Comment:
Worried about the wording here. The names are pretty short, and people may not know what "Intense Text Format" means. Should we do something more like `Display "intense" text in...` `...the normal font` `a bold font` `...bright colors` `...both bold font and bright colors`?

Should we get more technical and mention SGR 1?

Reply:
@cinnamon-msft what do you think? this is a button for users who "know what they're doing"

Reply:
I'm gonna guess that .1% of our users will know what SGR 1 is. I be 1% of the folks that upvoted #109 in the first place do - people just want bold. 

I'm not a wordsmith, [I'm just a lowly engineer](https://memegenerator.net/img/instances/400x/74828021.jpg)

Reply:
As a point of comparison, Mintty has this option as a dropdown list with the label "Show bold", and with the choices "as font", "as color", or "as font & as color". XTerm just as the option to toggle the bold fonts (at least in the UI) with a checked menu item labelled "Bold Fonts". And Gnome Terminal just as the option to toggle the bright colors with a checkbox labelled "Show bold text in bright colors".

My recommendation would be something similar to what Mintty has, but with a little more detail, e.g.:
- Show bold with brighter colors
- Show bold with a heavier font weight
- Show bold with both font weight and color

I think most people that are going to want to mess with this know that it's the "bold" functionality that they're tweaking. And what we're trying to differentiate is whether it affects the font or the color (or both).

Reply:
@j4james I appreciate that! Thanks. We're gonna merge this as is to get the feature into selfhost/test builds and work on the wording from there. :)

Reply:
(we're gonna work on the wording in post)